Database/database.py

```diff
@@ -0,0 +1,95 @@
+import json
+import sqlite3
+import random
+
+# connect to the database
+conn = sqlite3.connect("Database/allwords")
+cursor = conn.cursor()
+
+def generate_table():
+# load the JSON data from the file
+  with open("Database/dictionary.json", "r") as f:
+    data = json.load(f)
+
+  # create a table to store the data
+  conn.execute("""
+  CREATE TABLE IF NOT EXISTS all_words (
+    key TEXT,
+    word TEXT,
+    points INTEGER
+  )
+  """)
+  conn.execute("""
+  CREATE TABLE IF NOT EXISTS word_info (
+    key TEXT,
+    required_letter TEXT,
+    total_points INTEGER
+  )
+  """)
+
+  # insert the data into the table
+  for key, values in data.items():
+      required_letter = values[0]
+      total_points = values[1]
+      conn.execute("""
+      INSERT INTO word_info (key, required_letter, total_points)
+      VALUES (?,?,?)
+      """, (key, required_letter, total_points, ))
```

Comment:
Lines 37 and 43 - is the comma after total_points (37) and points (43) required to be there? I was looking around online and I saw that the values that are added don't have that last comma inside the parentheses

Reply:
They don't need to be nor is there a downside. Also, most of this code won't be run since the DB is already created and the values are already inserted.